leading-teams/peer-level-meeting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Peer-level meeting
+meta_description: >
+  Nebulab team leads have a weekly discussion in which they share problems and
+  solutions. Read our Playbook to learn how we keep a shared team culture and
+  support each other!
+---
+
+Each client has its own story and unique challenges, in order to share problems,
```

Comment:
its -> their

Reply:
Also, I think you should put a full stop after "challenges" and start a new sentence.